test/functional/cmdLineTests/jitserver/jitserverArgumentTesting.xml

```diff
@@ -89,4 +90,16 @@
 		<output type="failure" caseSensitive="yes" regex="no">JITSERVER NO LONGER EXISTS</output>
 	</test>
 
+	<test id="Test SSL">
+		<command>bash $SCRIPPATH$ $TEST_RESROOT$ $TEST_JDK_BIN$ "$DEFAULT_JITSERVER_OPTIONS$" "$ENABLE_JITSERVER$ $JITSERVER_VERBOSE$ $JITSERVER_SSL$" false</command>
+		<output type="success" caseSensitive="no" regex="yes" javaUtilPattern="yes">(java|openjdk|semeru) version</output>
+		<output type="required" caseSensitive="no" regex="no">JITServer Client Mode.</output>
+		<output type="required" caseSensitive="no" regex="no">Successfully initialized SSL context</output>
+		<output type="success" caseSensitive="no" regex="no">Connected to a server</output>
```

Comment:
I am not familiar with the testing infra. Should we change "success" to "required" here?
The client can still show `Successfully initialized SSL context` and still not connect successfully to the server. We must see both "Successfully initialized SSL context" and "Connected to a server" to guarantee a successful connection with SSL.

Reply:
I have now changed the test conditions in the new change.
@dsouzai would you know when we should use `"success"` vs `"required"` for the test conditions?